frontend/src/pages/Chats.tsx

```diff
@@ -1,9 +1,83 @@
 import TopAppBar from '../components/TopAppBar';
+import { useEffect, useState } from 'react';
+
+import { StreamChat } from 'stream-chat';
+import {
+  Chat,
+  Channel,
+  Window,
+  ChannelHeader,
+  MessageList,
+  MessageInput,
+  Thread,
+  LoadingIndicator,
+  ChannelList,
+} from 'stream-chat-react';
+
+import 'stream-chat-react/dist/css/index.css';
+import './Chats.css';
```

Comment:
Rather than using CSS, can we use Material UI styling to keep it consistent with the rest of the codebase? As is done [here](https://github.com/Siyer2/uni-connector/blob/3a78292ba792c0eb832c7f08d23655ee3e82985a/frontend/src/components/TopAppBar.tsx#L16), we can add it to the actual component

Reply:
For the imported StreamChat components, they already come with CSS container classes that inject basic display styling with `stream-chat-react/dist/css/index.css` - did you want me to use Material UI styling with these components?

As for `./Chats.css`, that is for changing the `height` property of one of the default containers that the StreamChat component automatically comes with - I'm not exactly sure how to change that into inline styling (adding an additional field of `style` with props creates an error.

Reply:
Actually, don't worry about migrating away from CSS. I'm getting a colleague of mine whose a frontend wiz (I'm certainly not 😅) to help us make the frontend best practice. One of the things he wants to do is use more CSS - so ignore this comment.